hw03/WebContent/inventory.jsp

```diff
@@ -0,0 +1,36 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+    <%@ taglib uri = "http://java.sun.com/jsp/jstl/core" prefix = "c" %>
+    <%@ taglib uri = "WEB-INF/admin-header.tld" prefix = "cs3220" %>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<link rel="stylesheet" href="<c:url value='/app.css' />">
+<title>Existing Inventory</title>
+</head>
+<cs3220:header title="The Good Food"/>
+<body bgcolor="lightgray">
+
+<h2>Existing Food Items</h2>
+
+	<table>
+		<tr>
+			<th>Created</th>
+			<th>Item</th>
+			<th>Actions</th>
+		</tr>
+		 <c:forEach items="${foods}" var="food">
+		<tr>
+			<td><%=new java.util.Date() %></td>
+			<td><center><img src ='${food.imageUrl}' alt='image' style='width:50px;height:50px' /><br />${food.name}</center></td>
+			<td><button>Delete</button></td>
```

Comment:
delete button isn't working. This delete button will not send the request containing which item you are deleting.